docs/infra/push-access-blocker-decision.md

```
@@ -0,0 +1,145 @@
+
+**Task:** AIWO-042
+**Date:** 2026-07-13
+**Status:** Resolved (workaround adopted, no org-admin action required)
+
+## Symptom
+
+Any direct push to `origin` (`https://github.com/Runfusion/Fusion.git`) from the
+`ischindl` git/GitHub identity fails, including for a plain branch push (not just
+`main`):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)